CHANGELOG.md

```diff
@@ -11,7 +11,8 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 - The transaction receipts for smart contract publish transactions now indicate
   a result of `(err none)` if the top-level code of the smart contract contained
-  runtime error. Fixes issue #3154.
+  runtime error and include details about the erro in the `vm_error` field of
```

Comment:
```suggestion
  runtime error and include details about the error in the `vm_error` field of
```

Reply:
Oops! Fixed in 78b9a8c504ed97880d94c232f762608ee7e296d0.